### ISSUE TITLE
`use Pandoc NUMBER` to check for executable version won't work

### ISSUE THREAD

**Issue by bpj** [nichtich/Pandoc-Wrapper#7]
`use Pandoc NUMBER` to check for executable version won't work.

It will cause perl to check for version NUMBER of Pandoc.pm!

https://github.com/nichtich/Pandoc-Wrapper/blob/master/lib/Pandoc.pm#L229

So the safest route is probably anyway to pass `version` and `bin` parameters in a hashref both to importer and constructor. Many modules use hashrefs in the import statement to pass named options. It is to be regarded as perfectly normal. I would allow passing args as an array ref as well, since they aren't imports in the non-OO sense.

**Comment by nichtich** [comment]
I added a fix but there is another bug comparing version numbers with [module version](https://metacpan.org/pod/version) and "old style" vs "new style" version numbers:

```
$ perl -Ilib -E 'use Pandoc 1.9'
pandoc 1.9 required, only found 1.16.0.2
```

**Comment by nichtich** [comment]
Have you tried that it does not work? The import function should take care of it. We could add a note to the documentation how to check `$Pandoc::VERSION` if actually needed.

On 22. Oktober 2016 15:57:53 MESZ, Benct Philip Jonsson notifications@github.com wrote:

> `use Pandoc NUMBER` to check for executable version won't work.
> 
> It will cause perl to check for version NUMBER of Pandoc.pm!
> 
> https://github.com/nichtich/Pandoc-Wrapper/blob/master/lib/Pandoc.pm#L229
> 
> So the safest route is probably anyway to pass `version` and `bin`
> parameters in a hashref both to importer and constructor. Many modules
> use hashrefs in the import statement to pass named options. It is to be
> regarded as perfectly normal. I would allow passing args as an array
> ref as well, since they aren't imports in the non-OO sense.
> 
> ## 
> 
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/nichtich/Pandoc-Wrapper/issues/7

## 

Diese Nachricht wurde von meinem Android-Mobiltelefon mit K-9 Mail gesendet.

**Comment by nichtich** [comment]
Fixed in release 0.3.1
